index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO: SETUP AND MOBILE VERSION SKELETON</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <header>
+            <div id="toolbar">
+                <nav class="left">
+                    <img src="Time.png" alt="">
+                </nav>
+                <nav class="right">
+                    <img src="Cellular.png" alt="Cellular">
+                    <img src="Wifi.png" alt="Wifi">
+                    <img src="Battery.png" alt="battery">
+                </nav>
+                <nav>
+                    <div class="toolbar1">
+                        <img src="Logo.png" alt="Logo">
+                    </div>
+                    <div class="toolbar2">
+                        <img src="Union.png" alt="Menu">
+                    </div>
+                </nav> 
+            </div>
+        </header>
+        <main class="name">
+            <p>I'm Kelvin Glad to see you!</p>
+        </main>
+        <div class="text">
+            <p>I'm a software developer! I can help you build a product,
+                feature or website. Look through someof my work and experience!
+                If you like what you see and have a project you need coded,
+                don't hesitate to contact me.
+            </p>
+        </div>
+        <nav id="lets-connect">
+            <h5>LET'S CONNECT</h5>
+        </nav>
+        <div class="icons">
+            <img src="icons.png" alt="Social Media Icons">
+        </div>
+    </div>
```

Comment:
- [ ] You have done amazing work but you still need to fix your design to match the given template. Your icons, logo, and background image aren't showing as expected. I will suggest you take a look at your image path to verify they are correct in order to render the images. 

Your Design | Figma Design |
:----------------:|:-------------------:|
![a2](https://user-images.githubusercontent.com/39523312/171420163-980bcabf-5859-4704-942d-fb7382a4e854.png)| ![a1](https://user-images.githubusercontent.com/39523312/171420181-52b084db-4b6e-4f64-b351-c4f1778624c9.png)

Reply:
@zieeco the font family is not available in my vs code. That's why I used an alternative one. What should I do maybe?